src/components/Checkbox/Checkbox.tsx

```diff
@@ -15,6 +16,6 @@ const defaultProps = {
 export const Checkbox = withStylable<CoreCheckboxProps, CheckboxProps>(
   CoreCheckbox,
   style,
-  ({skin, size}) => ({[size]: true, [skin]: true}),
+  ({skin, size, active}) => ({[size]: true, [skin]: true}),
```

Comment:
ur not using active

Reply:
I added it but decided to remove and just use the native one. I just forgot to remove this instance.